src/utils/businessHours.ts

```diff
@@ -147,14 +151,14 @@ export const isChannelInBusinessHours = async (
 };
 
 export async function getNextAvailableBusinessHourWindow(
-  team,
+  productArea,
   momentTime
 ): Promise<BusinessHourWindow> {
-  let offices = await getOffices(team);
+  let offices = await getOffices(productArea);
   if (offices.length === 0) {
-    offices = await getOffices('Team: Open Source');
+    offices = await getOffices('Product Area: Open Source');
```

Comment:
This product area won't exist, right? Do we need to hard code offices for this case instead?

Reply:
Yep we can, technically I think our team or Support should handle all cases when Product Area doesn't have a clear product owner